ui/src/pages/Install/index.tsx

```diff
@@ -131,11 +131,35 @@ const Index: FC = () => {
     },
   });
 
+  const updateFormData = (params: FormDataType) => {
+    if (step === 2) {
```

Comment:
Monitoring changes in value of db_type is more accurate

 if (params.db_type?.value !== formData.db_type.value) {
  ...
}

Reply:
Thanks, It make sense. The latest changed has been updated at https://github.com/apache/incubator-answer/pull/1016/commits/b18016fd97f3112860dd4beb5ae08a177b62b128